### PR TITLE
Expose IErrorHandler and add test case.

### PR DESCRIPTION
**Pull request by imcarolwang**
For issue #5561.

### CODE REVIEW

File: src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs [hunk 1]
```diff
@@ -1834,6 +1834,7 @@ internal ClientRuntime() { }
         public System.Type ContractClientType { get { return default; } set { } }
         public string ContractName { get { return default; } }
         public string ContractNamespace { get { return default; } }
+        internal bool EnableFaults { get { return default; } set { } }
```
Comment (mconnew) [inline src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs:1837]: I'm trying to understand why you added this to the ref assembly as it's internal?

**Comment by mconnew** [comment]
Everything looks good except the question around the internal EnableFaults. With the recent changes I made to support netstandard2.0 and netfx, we have multiple ref assemblies. When you rebase, only add to the netcoreapp ref assembly, we aren't updating the netstandard2.0 reference assembly beyond what it currently is as it's only there to support those using that capability from the 4.10.x packages.

File: src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs (hunk 1 above)
Comment (imcarolwang) [reply src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs:1837]: I think the request at https://github.com/dotnet/wcf/issues/5561#:~:text=time.%0A%40imcarolwang%2C-,could%20you%20expose%20IErrorHandler%2C%20ChannelDispatcher.ErrorHandlers%2C%20and%20ClientRuntime.EnableFaults,-and%20add%20tests%20for suggests adding `ClientRuntime.EnableFaults`. It is internal in the src implementation https://github.com/dotnet/wcf/blob/4e6e777613d380d9101969d15c8f08fc543a22fd/src/System.ServiceModel.Primitives/src/System/ServiceModel/Dispatcher/ClientRuntime.cs#L225, so I brought the same to the ref project.

Would you recommend that we update it?

Comment (imcarolwang) [reply src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs]: @mconnew Thank you for letting me know. I’ve updated the PR to remove the exposure of "EnableFaults." Following your suggestion, I’ve rebased the branch and made changes only to the netcoreapp reference assembly by modifying the System.ServiceModel.Primitives.Netcoreapp.cs file. Would you mind taking another look when you have a chance to confirm if everything looks good now?

**Review by mconnew — APPROVED** [review]

**Pull request merged**